using-mobile-extensions/mobile-core/configuration/configuration-api-reference.md

```diff
@@ -64,63 +69,66 @@ Alternatively, you may also place the Launch environment ID in your iOS project'
 
 ## Programmatic updates to configuration
 
-You can also update the configuration programmatically by passing configuration keys and values to override existing configuration. 
+You can also update the configuration programmatically by passing configuration keys and values to override existing configuration.
 
 {% hint style="info" %}
 Keys not found on the current configuration are added when this method is followed. Null values are allowed and will replace existing configuration values.
 {% endhint %}
 
 {% tabs %}
 {% tab title="Android" %}
-#### Java
 
 ### updateConfiguration
 
-#### Syntax <a id="syntax-3"></a>
+#### Syntax
 
 ```java
-void updateConfiguration(final Map configMap);
+public static void updateConfiguration(final Map configMap);
 ```
 
-#### Example <a id="example-3"></a>
+#### Example
 
 ```java
 HashMap<String, Object> data = new HashMap<String, Object>();
 data.put("global.ssl", true);
-data.put("global.timezone", "PDT");
-data.put("global.timeoneOffset", -420);
 MobileCore.updateConfiguration(data);
 ```
 {% endtab %}
 
 {% tab title="iOS" %}
-#### Objective-C
-
 ### updateConfiguration
 
+{% hint style="success" %}
+Starting in ACPCore 2.0.3 for iOS, any key provided via __updateConfiguration__ should not have a prefix.  The configuration extension may modify the key provided to be correct for the corresponding `build.environment` in which the app is running.  For more information, see [Environment-aware Configuration Properties](README.md#environment-aware-configuration-properties).
+{% endhint %}
+
 #### Syntax
 
 ```objectivec
 + (void) updateConfiguration: (NSDictionary* __nullable) config;
 ```
-
 #### Example
-
+###### Objective-C
 ```objectivec
-NSMutableDictionary *updatedConfig = [NSMutableDictionary dictionary]; 
-[contextData setObject:@"newrsid" forKey:@"analytics.rsid"]; 
+NSDictionary *updatedConfig = @{@"global.ssl":@YES};
 [ACPCore updateConfiguration:updatedConfig];
 ```
+###### Swift
+```swift
+let updatedConfig = ["global.ssl":true]
+ACPCore.updateConfiguration(updatedConfig)
+```
+
 {% endtab %}
 {% endtabs %}
 
-### Using a bundled file configuration
+## Using a bundled file configuration
 
-You can also choose to include a bundled JSON configuration file in your app package to replace or complement by using the [Launch environment ID](./#launch-environment-id) approach.
+You can also choose to include a bundled JSON configuration file in your app package to replace or complement by using the [Configure with Launch App ID](./#configure-with-launch-app-id) approach.
```

Comment:
This sentence is confusing. What is "replace or complement" referring to? (That is, what is being replaced or complemented?)